.storybook/main.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    webpackFinal: async (config, { configType }) => {
+        const rules = config.module.rules;
+        const fileLoaderRule = rules.find(rule => rule.test.test('.svg'));
+        fileLoaderRule.exclude = /\.svg$/;
+
+        rules.push({
+            test: /\.svg$/,
+            use: ["@svgr/webpack"],
+        });
+
+        return config;
+    },
+    "stories": [
```

Comment:
Strange to see simple keys in quotes. But I think this is problem with formatter.